components/atlas-uvs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aframe-atlas-uvs-component",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

Comment:
Bumping the component version is not needed. Done when cutting a new release.

Reply:
How to specify version numbers in docs if this is auto-incremented? Just guess that it will be 0.2.2? Also how to specify a major version change (3.0) instead of a minor version change (2.2) since this is a breaking change?

Reply:
Ok reverted back